src/commands/ScheduledElements.php

```diff
@@ -0,0 +1,174 @@
+
+        // Print info
+        ConsoleHelper::output(sprintf("> Expired Entries: %d", count($expired)));
+        ConsoleHelper::output(sprintf("> Published Entries: %d", count($published)));
+        ConsoleHelper::output(sprintf("> Range: %s to %s", $lastCheck, $now));
+
+        if (!count($entries)) {
+            return ExitCode::OK;
+        }
+
+        $this->fireEvent($published, Entry::STATUS_PENDING);
+        $this->fireEvent($expired, Entry::STATUS_LIVE);
+
+        $this->drawTable($entries);
+
+        return ExitCode::OK;
+    }
+
+    /**
+     * @param array $entries
+     */
+    protected function drawTable(array $entries)
+    {
+        $rows = [];
+
+        foreach ($entries as $entry) {
+            /** @var Entry $entry */
+            $postDateString   = $entry->postDate ? $entry->postDate->format(self::DATE_FORMAT) : '-';
+            $expiryDateString = $entry->expiryDate ? $entry->expiryDate->format(self::DATE_FORMAT) : '-';
+            $rows[]           = [$entry->title, $postDateString, $expiryDateString, $entry->id, $entry->slug];
+
+            // Also write to stdout for cloud logging
+            Craft::warning("K8 CronJob revalidating: $entry->slug", "REVALIDATE_STATUS");
+        };
+
+        echo Table::widget([
+            'headers' => ['Title', 'PostDate', 'ExpiryDate', 'ID', 'Slug'],
+            'rows'    => $rows,
+        ]);
+    }
+
+    /**
+     * @param array  $elements
+     * @param string $previousStatus
+     */
+    protected function fireEvent(array $elements, $previousStatus = '')
+    {
+        if (count($elements) === 0) {
+            return;
+        }
+
+        // statusBeforeSave
+        foreach ($elements as $element) {
+            Event::trigger(
+                ElementStatusEvents::class,
+                ElementStatusEvents::EVENT_STATUS_CHANGED,
+                new StatusChangeEvent([
+                    'element'          => $element,
+                    'previousStatus' => $previousStatus
+                ])
+            );
+        }
+    }
+
+
+    /**
+     * @param $rangeStart
+     * @param $rangeEnd
+     *
+     * @return array
+     */
+    protected function getPublishedEntries($rangeStart, $rangeEnd): array
+    {
+        $entries = (Entry::find()
+            ->where(['between', 'postDate', $rangeStart, $rangeEnd])
+            ->withStructure(false)
+            ->orderBy(null)
+            ->status('live')
+        )->all();
+
+        return array_filter($entries, function (Entry $item) {
+            $diffInSeconds = abs($item->postDate->getTimestamp() - $item->dateUpdated->getTimestamp());
+            return ($diffInSeconds > 60);
+        });
+    }
+
+    /**
+     * @param $rangeStart
+     * @param $rangeEnd
+     *
+     * @return Entry[]
+     */
+    protected function getExpiredEntries($rangeStart, $rangeEnd): array
+    {
+        // TODO: Support Product and other Elements with expiryDate
+
+        return (Entry::find()
```

Comment:
Another query, basically the event command queries the database for entries that have changed their status since this plugin started keeping track of status changes.